pkg/codegen/go/gen.go

```diff
@@ -117,7 +117,10 @@ func Title(s string) string {
 }
 
 func tokenToPackage(pkg schema.PackageReference, overrides map[string]string, tok string) string {
-	mod := pkg.TokenToModule(tok)
+	return moduleToPackage(pkg, overrides, pkg.TokenToModule(tok))
+}
+
+func moduleToPackage(pkg schema.PackageReference, overrides map[string]string, mod string) string {
```

Comment:
Since we're here, might be good to whack in some doc comments with an example of what this does? I find that's one of the hardest things to keep swapping in in codegen.